src/SDKs/_metadata/cognitiveservices_data-plane_CustomImageSearch.txt

```diff
@@ -1,9 +1,9 @@
-2018-05-03 09:18:53 UTC
+2018-09-05 06:24:32 UTC
 
 1) azure-rest-api-specs repository information
 GitHub user: Azure
 Branch:      master
```

Comment:
Was this manually edited?
Was the REST spec PR merged?

Reply:
It was generated with a hack (looking at a different repo) to avoid waiting until the spec PR is merged. The spec PR is now merged. I can regenerate if needed.

Reply:
Please regenerate and check in all artifacts generated.
Please avoid using hacks in the future,, if you wish to generate SDK using a branch that has not been merged yet, the `Start-AutoRestCodeGeneration` cmdlet has parameters to allow that